### ISSUE TITLE
Implement 2.01 "3. Language neutral codelists"

### ISSUE THREAD

**Issue by Bjwebb** [IATI/IATI-Codelists#25]
This is section 3 on the thematic overview: http://support.iatistandard.org/entries/45539933-Thematic-content-agreed-by-IATI-Steering-Committee

http://support.iatistandard.org/entries/57958296-Version-2-01-Iteration-1-3-Language-Neutral-Code-lists

This applies to:
- http://dev.iatistandard.org/201/codelists/ActivityDateType/
- http://dev.iatistandard.org/201/codelists/GazetteerAgency/
- http://dev.iatistandard.org/201/codelists/OrganisationRole/
- http://dev.iatistandard.org/201/codelists/TransactionType/
- http://dev.iatistandard.org/201/codelists/Vocabulary/

**Comment by Bjwebb** [comment]
I suggest that we simply rename the codes (ie. delete the old ones), instead of deprecating. http://support.iatistandard.org/entries/51310806-Delete-don-t-just-deprecate-codes-in-2-01

**Comment by Bjwebb** [comment]
Following discussion with @bill-anderson I'm going to implement this in Iteration 1 by replacing (deleting the old codes), instead of deprecating.

**Comment by caprenter** [comment]
See: http://support.iatistandard.org/entries/57958296-Version-2-01-Iteration-1-3-Language-Neutral-Code-lists
for an update.

Seems to me that there are 2 more changes to make:
- [x] "There was consensus at the Montreal TAG that "accountable" should be removed from the list" https://github.com/IATI/IATI-Codelists/issues/33
- [x] "Change name to SectorVocabulary" (from Vocabulary)

**Comment by caprenter** [comment]
Removing 'accountable' from the OrgansiationRole list will mean renaming the codes on that list as set in this commit: https://github.com/IATI/IATI-Codelists/commit/7c6b2228b3e1483aec431ba3194684441a6eaa32 unless we want to start the codes at '2'.

**Comment by caprenter** [comment]
We want to follow the same conventions in the codelist xml as adopted in the activity schema for handling translations - this is yet to be agreed, but will mean altering the underlying codelist schema.

**Comment by bill-anderson** [comment]
Agree we should renumber codes

**Comment by Bjwebb** [comment]
I've now removed Accountable and renumbered [OrganisationRole](http://dev.iatistandard.org/201/codelists/OrganisationRole/) and done the rename to [SectorVocabulary](http://dev.iatistandard.org/201/codelists/SectorVocabulary/)

A couple of questions about SectorVocabulary:
- I numbered this in the order the codes were previously on the list. This makes the DAC codes `3` and `4`. Should these be renumbered to be `1` and `2` as they are the recommended options?
- [policy-marker/@vocabulary](http://dev.iatistandard.org/201/activity-standard/iati-activities/iati-activity/policy-marker/#iati-activities-iati-activity-policy-marker-vocabulary) uses the  SectorVocabulary codelist (in 1.04 it uses the Vocabulary codelist). Should it, or should there be a PolicyMarkerVocabulary codelist?
